start/generate.bash

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+
+if [[ -d build ]] ; then
+    echo "Please delete ${PWD}/build directory first."
+    exit
+fi
+
+# Clone dvc-get-started
+git clone https://github.com/iterative/dvc-get-started
+
+GIT_TAGS=$(git -C dvc-get-started tag --list)
+
+for TAG in ${GIT_TAGS} ; do
```

Comment:
```suggestion
git -C dvc-get-started tag --list | while read -r TAG ; do
```

Reply:
I should learn to use `for` less. Thanks.

Reply:
Most of this is related with writing `zsh` more than `bash`. I probably translated this from `for TAG in "${(@f)(git tag --list)}"`